src/command_line_parser.cc

```diff
@@ -824,71 +824,90 @@ CLParser::ParseCommandLine(int argc, char** argv)
   argc_ = argc;
   argv_ = argv;
 
+  constexpr int long_option_idx_base{256};
```

Comment:
We started having so many long options that they indices were overlapping with short options (e.g. ascii `?` is integer 63), so I created a base value beyond all ascii values to start the long options from

Reply:
Sure, the options deserve a deeper dive/redesign.